src/app/playground/components/playground-result/playground-result.component.scss

```diff
@@ -0,0 +1,15 @@
+:host {
```

Comment:
I don't think this is needed.

Reply:
It is actually very important  because I wanted the child flex to grow to the maximum based on the parent container.

That's how it looks without styling the :host

![image](https://user-images.githubusercontent.com/6767322/89113103-065f0300-d49f-11ea-986b-d47ca19049cd.png)

And with the `:host { @apply flex flex-col flex-grow h-full; }`

![image](https://user-images.githubusercontent.com/6767322/89113113-2e4e6680-d49f-11ea-9254-89a3484173d5.png)

Reply:
The thing is you set `:host` to have `h-full` just to have the immediate child `div` with `flex-grow` to expand the full height. If you just put `h-full` on the div, it'd work the same. However, if you want to use `:host`, I'd suggest getting rid of the immediate `div`

Reply:
Oh yeah, that's true. Thanks for your comment. I pushed the changes, please take a look.